Earthfile

```diff
@@ -1,27 +1,30 @@
 VERSION 0.6
+FROM python:3.8
+WORKDIR /contrib
 
-prepare:
-  FROM python:3.8
-  WORKDIR /contrib
+src-files:
   COPY electricitymap ./electricitymap
   COPY parsers ./parsers
   COPY validators ./validators
   COPY config ./config
   COPY scripts ./scripts
   COPY __init__.py ./__init__.py
   COPY pyproject.toml .
+  SAVE ARTIFACT .
+
+prepare:
+  FROM +src-files
   RUN pip install poetry==1.1.12
   RUN apt-get update && apt-get install -y python3-opencv tesseract-ocr tesseract-ocr-jpn tesseract-ocr-eng libgl1
   RUN poetry config virtualenvs.create false
   RUN poetry install -E parsers -E validators
 
-test:
+build:
```

Comment:
does the `SAVE ARTIFACT .` from the src-files get transfered all the way to the build step?

Reply:
Nope, the artifact is only saved here. For now, we don't really have a need to save artifacts from the other steps. We could add a `SAVE ARTIFACT .` in the build step if need to depend on that in the future. (e.g. if we want to expose built wheels or something like that)